pwa/app/routes/event.$eventKey.tsx

```diff
@@ -96,17 +98,27 @@ async function loadData(params: Route.LoaderArgs['params']) {
     });
   }
 
-  const [event, matches, alliances, rankings, awards, teams, teamMedia, coprs] =
-    await Promise.all([
-      getEvent({ eventKey: params.eventKey }),
-      getEventMatches({ eventKey: params.eventKey }),
-      getEventAlliances({ eventKey: params.eventKey }),
-      getEventRankings({ eventKey: params.eventKey }),
-      getEventAwards({ eventKey: params.eventKey }),
-      getEventTeams({ eventKey: params.eventKey }),
-      getEventTeamMedia({ eventKey: params.eventKey }),
-      getEventCopRs({ eventKey: params.eventKey }),
-    ]);
+  const [
+    event,
+    matches,
+    alliances,
+    rankings,
+    awards,
+    teams,
+    teamMedia,
+    coprs,
+    colors,
+  ] = await Promise.all([
+    getEvent({ eventKey: params.eventKey }),
+    getEventMatches({ eventKey: params.eventKey }),
+    getEventAlliances({ eventKey: params.eventKey }),
+    getEventRankings({ eventKey: params.eventKey }),
+    getEventAwards({ eventKey: params.eventKey }),
+    getEventTeams({ eventKey: params.eventKey }),
+    getEventTeamMedia({ eventKey: params.eventKey }),
+    getEventCopRs({ eventKey: params.eventKey }),
+    getEventColors({ eventKey: params.eventKey }),
```

Comment:
Doesn't have to be done in this PR, but eventually we should probably look at not blocking the whole page load on every single one of these requests (especially 3rd party ones like colors).